module/x/gravity/keeper/invariants_test.go

```diff
@@ -65,14 +67,15 @@ func TestModuleBalanceUnbatchedTxs(t *testing.T) {
 
 	// Ensure an error is returned for a mismatched balance
 	oneVoucher, _ := types.NewInternalERC20Token(sdk.NewInt(1), myTokenContractAddr)
```

Comment:
This test invalidated the module balance so I used your self-healing checkImbalancedModule function to make the test consistent with the rest of them.